Dockerfile

```diff
@@ -39,7 +39,8 @@ COPY . .
 COPY --from=frontend-builder /cpmonitor/cpmonitor/static/css ./cpmonitor/static/css
 COPY --from=frontend-builder /cpmonitor/node_modules/@tabler/core/dist ./node_modules/@tabler/core/dist
 COPY --from=frontend-builder /cpmonitor/node_modules/jquery/dist ./node_modules/jquery/dist
-RUN /venv/bin/python manage.py collectstatic --no-input -v 2 --settings=config.settings.container
+RUN DJANGO_SECRET_KEY="dummy" DJANGO_CSRF_TRUSTED_ORIGINS="dummy" DJANGO_DEBUG="False" \
```

Comment:
Why is this needed?

Reply:
I moved the values for those variables from the config/settings python modules to the docker .env files, so python would complain about missing values if I don't pass them in here.
It's not very elegant, but it was the easiest way I could think of without having multiple config/settings modules.